diff --git a/wrappers/python/setup.py b/wrappers/python/setup.py
index f5de33531..da3ffd36a 100644
--- a/wrappers/python/setup.py
+++ b/wrappers/python/setup.py
@@ -166,6 +166,7 @@ def buildKeywordDictionary(major_version_num=MAJOR_VERSION_NUM,
     (especially on recent GPUs) that make it truly unique among simulation codes.
     """
     setupKeywords["extras_require"]         = {"cuda12": f"openmm-cuda=={setupKeywords['version']}.12", "cuda11.2": f"openmm-cuda=={setupKeywords['version']}.11.2", "cuda11.8": f"openmm-cuda=={setupKeywords['version']}.11.8"}
+    setupKeywords["install_requires"]  = ["numpy<2"]
 
     define_macros = [('MAJOR_VERSION', major_version_num),
                      ('MINOR_VERSION', minor_version_num)]

